libs/design-system/src/molecules/IFramedSidePanel/IFramedSidePanel.tsx

```diff
@@ -117,6 +124,14 @@ export const IFramedSidePanel = ({
     </>
   );
 
+  const closeContent = <OpenExtensionBubble onClick={onOpenCallback} />;
+
+  const leftPosition = isOpen
+    ? Math.min(Math.max(10, sidePanelLeftPosition), window.innerWidth - 356)
+    : window.innerWidth - 116;
+
+  const topPosition = isOpen ? null : { top: '100px' };
```

Comment:
This doesn't work. If you resize the panel, the icon disappears. Actually, the problem is similar for the iFramed. You should add a listener on resize, something like:

```
    const resizeHandler = () => {
      setSidePanelLeftPosition((prev) => Math.min(prev, window.innerWidth - 356));
    };
```

To make sure it stays in the window. Careful, my example here works for downsizing but not for upsizing.

Adn TBH, I think the logic shouldn't live here. This component ends displaying something that it's not.

Reply:
Implemented a solution to handle window resize events. However, the logic is still in the `IFramedSidePanel` component so far.